src/browser/modules/Stream/CypherFrame/CodeView.test.tsx

```diff
@@ -84,10 +84,11 @@ describe('CodeViews', () => {
           summary: {
             resultAvailableAfter: neo4j.int(5),
             resultConsumedAfter: neo4j.int(5)
-          },
-          maxRows: 100,
-          records: [{ res: 'xx3' }]
-        }
+          } as any,
```

Comment:
Do we need `as any` in such places? I don't see TSlint warning without this.

Reply:
yes I get typescript errors otherwise. Probably we need to sync our settings then.

Reply:
@eijawerner - my bad. Checked with your branch and it should be typed anyway.